test/lit/passes/log-execution_arg.wast

```diff
@@ -0,0 +1,146 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; Test the option to provide the module name as an argument.
+;; RUN: foreach %s %t wasm-opt --log-execution=foo -S -o - | filecheck %s
+
+(module
+  ;; CHECK:      (type $0 (func))
+
+  ;; CHECK:      (type $1 (func (result i32)))
+
+  ;; CHECK:      (type $2 (func (param i32)))
+
+  ;; CHECK:      (import "env" "func" (func $import))
+  (import "env" "func" (func $import))
+  ;; CHECK:      (import "foo" "log_execution" (func $log_execution (param i32)))
+
+  ;; CHECK:      (func $nopp
+  ;; CHECK-NEXT:  (call $log_execution
+  ;; CHECK-NEXT:   (i32.const 0)
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT:  (nop)
+  ;; CHECK-NEXT: )
+  (func $nopp
+    (nop)
+  )
+  ;; CHECK:      (func $intt (result i32)
```

Comment:
I think it's ok to stop this test here. The first 25 lines show that it uses "foo" and that it emits a valid call to `log_execution`.

Reply:
(That is, it doesn't need to be as comprehensive as the other test, as the focus here is just on the argument to the pass, which lines 1-25 test.)

Reply:
are you suggesting to minimize the test?

Reply:
done

Reply:
Yes, exactly.